_guides/legion/bios/bios-recovery.md

```diff
@@ -1,32 +1,66 @@
 ---
 layout: wiki
-title: Bios Recovery
-description: "Recovering bricked BIOS in Legions without a backup ROM"
+title: BIOS Recovery
+description: "Recovering bricked BIOS in Legions using a SPI"
 parent: Bios
 grand_parent: Legion
-contributors:
-discord: ['LABØ!#5022']
+contributors: ['labodj'] 
 ---
 
-# Recovering bricked BIOS in Legions without a backup ROM
+# Recovering bricked BIOS in Legions using a SPI
 
 ## My Legion Laptop BIOS is screwed up and I don't have a BIOS backup, how can I try to restore it?
 
-Basically a 3.3V SPI programmer like the CH341A is needed and also knowledge about it is required. This [video](https://www.youtube.com/watch?v=4qX2zihB6UE){:target="_blank"} should help to understand how the programmer works, if you still cant figure out how this programmer works, then this guide isn't for you. 
+A 3.3V SPI programmer like the CH341A is needed and also knowledge about it is required. This [video](https://www.youtube.com/watch?v=4qX2zihB6UE){:target="_blank"} should help to understand how the programmer works, if you still can't figure out how SPI works, then this guide isn't for you.
```

Comment:
I know this has not been changed from the og draft, but wouldn't it be better to embed the video after the paragraph? 

The embed code from YouTube should work.

Reply:
Thanks for the suggestion, YT video ha been embedded in b0300465b1dab7bfcb693be47ce5f725519a9080